### PR TITLE
Audius integration

### LINKED ISSUE #16: Customize environment
https://github.com/feiss/aframe-environment-component

### PR DESCRIPTION
**Pull request by barrasso**
- Add jukebox
- Add audio stream and track details
- Add custom environment and lighting (Closes #16)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,8 +62,20 @@
         room: openverse;
         "
       >
-        <a-entity environment="preset: japan"></a-entity>
```
Comment (laughinglemur) [inline index.html:65]: #tokyo

File: index.html [hunk 2]
```diff
@@ -62,8 +62,20 @@
...
+        <a-entity environment="
+          dressing: mushrooms;
+          dressingScale: 8;
+          dressingColor: #8C14FC;
+          dressingAmount: 100;
+          groundColor: #333; 
+          groundTexture: walkernoise; 
+          lightPosition: 1 -1 -1;
+          seed: 6969;"
```
Comment (laughinglemur) [inline index.html:73]: ;)

File: index.html [hunk 3]
```diff
@@ -211,44 +224,80 @@
...
+          <a-plane
+            position="0 0.20 0.01"
+            scale="0.9 0.5 0"
+            material="shader: flat; transparent: true; opacity: 0.0;"
+          >
+            <a-entity position="0 0 1500">
+              <a-entity
+                id="player-details-description"
+                position="0 0.40 1"
+                text="value: you are staring at:; align: center; color: white; width: 3"
```
Comment (laughinglemur) [inline index.html:236]: suggestion to change to "looking", instead of "staring"?

File: index.html [hunk 4]
```diff
@@ -211,44 +224,80 @@
...
+          gltf-model="models/jukebox/scene.gltf"
+          animation__mouseenter="property: scale; to: 1.2 1.2 1.2; dur: 300; startEvents: mouseenter"
+          animation__mouseleave="property: scale; to: 1 1 1; dur: 300; startEvents: mouseleave"
+        >
+          <a-entity light="type: ambient; color: #8C14FC"></a-entity>
+          <a-sound
+            id="jukebox-audio"
+            src=""
+            autoplay="true"
+            positional="true"
```
Comment (laughinglemur) [inline index.html:276]: 👍

**Review by laughinglemur — APPROVED** [review]
LGTM!

File: index.html (hunk 2 above)
Comment (nazariyv) [reply index.html:73]: haha cheeky

File: index.html (hunk 3 above)
Comment (nazariyv) [reply index.html:236]: it was my doing. I have nothing against this proposal

File: index.html [hunk 5]
```diff
@@ -62,8 +62,20 @@
...
+          dressingAmount: 100;
+          groundColor: #333; 
+          groundTexture: walkernoise; 
+          lightPosition: 1 -1 -1;
+          seed: 6969;" 
+          ></a-entity>
+          <a-entity light="type: point; intensity: 1; distance: 100; decay: 4" position="0 10 5"></a-entity>
+
         <a-assets>
+          <img id="audius-logo" src="/images/audius-logo.png" />
```
Comment (nazariyv) [inline index.html:78]: we may want to use git lfs in the future

Comment (nazariyv) [reply index.html:78]: just checked the other files, which are quite large. I would move them all to git lfs

**Review by nazariyv — APPROVED** [review]
LGTM, but I would add git lfs for the large files (.jpeg and .png)

**Comment by microchipgnu** [comment]
> LGTM, but I would add git lfs for the large files (.jpeg and .png)

We also have .gltf files. Let's create an issue for that.

**Pull request merged**